### PR TITLE
BAEL-3142 Removing an element from an ArrayList

### PR DESCRIPTION
**Pull request by dcalap**

### CODE REVIEW

File: core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.baeldung.java.list;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class RemoveFromList {
```
Comment (rpvilao) [inline core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java]: fix indentation in the class

File: core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+                // Remove with an element
+                sports.remove("Baseball");
+
+                // Iterator remove method
+                Iterator<String> iterator = sports.iterator();
+                while (iterator.hasNext()) {
+                        if (iterator.next().equals("Boxing")) {
+                                iterator.remove();
+                        }
```
Comment (manojsharma27) [inline core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java]: Add a `break;` after line no 29. No need to iterate further once 'Boxing' is removed.

File: core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                }
+
+                // ArrayList removeIf method (Java 8)
+                for (int i = 0; i < sports.size(); i++) {
```
Comment (manojsharma27) [inline core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/java/list/RemoveFromList.java]: This loop is not required at all. As per comment at line 33, only the statement `sports.removeIf(p -> p.equals("Cycling"));` is enough.

**Comment by dcalap** [comment]
Totally agree with the changes. Updated code.

**Pull request merged**